src/patternfly/components/FormControls/form-controls.hbs

```diff
@@ -13,7 +13,7 @@
         {{#if id}}id="{{id}}"{{/if}}
```

Comment:
Extra return at the top of this file. Looks like indents are set to 8 spaces as well.